_posts/0000-01-21-githubteacher.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Making a miner change
```

Comment:
@githubteacher  Looks great.

Reply:
> This needs more work - Fix it 🍰 🚯

This is comedy gold, I don't know what you're talking about.